src/settings/SettingsModal.tsx

```diff
@@ -163,6 +177,33 @@ export const SettingsModal = (props: Props) => {
             <Button onPress={downloadDebugLog}>Download Debug Logs</Button>
           </FieldRow>
         </TabItem>
+        <TabItem
+          title={
+            <>
+              <SettingsIcon width={16} height={16} />
+              <span>Advanced</span>
+            </>
+          }
+        >
+          <InputSlider
+            label="Voice activation threshold"
+            children={
+              <>
+                <Button
+                  size="lg"
+                  variant="default"
+                  onPress={() => handleVoiceActivationTesting()}
```

Comment:
I am not sure why we're passing this this way, it should also probably be wrapped in `useCallback`